extension/src/setup/index.ts

```diff
@@ -300,11 +281,54 @@ export class Setup
     }
   }
 
-  public removeStudioAccessToken() {
-    return this.removeSecret(STUDIO_ACCESS_TOKEN_KEY)
+  public async removeStudioAccessToken() {
+    if (!this.getCliCompatible()) {
+      return
+    }
+
+    if (this.dvcRoots.length !== 1) {
+      const cwd = getFirstWorkspaceFolder()
+      if (!cwd) {
+        return
+      }
+      return this.internalCommands.executeCommand(
+        AvailableCommands.CONFIG,
+        cwd,
+        Flag.GLOBAL,
+        Flag.UNSET,
+        DvcConfigKey.STUDIO_TOKEN
+      )
+    }
+
+    const cwd = this.dvcRoots[0]
```

Comment:
[F] If there is only 1 dvc root in the workspace then we try to clear out the local and global configs. If the user has set the token into their `--system` config this won't work but I think this approach will get us close enough with the vast majority of users using the extension to manage their token.